Main2.cpp

```diff
@@ -152,11 +159,16 @@ int main()
         cout<<pop_grow<<endl;
         food_grow = land*1000;
         mili_grow = military_factory*25;
+        double x = double(city);
+        gdp = ((pow(1.1,x))*gdp); // Growth of GDP with City Growth
+        double y = double(military_factory);
+        gdp = ((pow(1.1,y))*gdp); // Growth of GDP with Military Factory growth
 
-        food_stores = (food_stores - population) + (food_grow);
+        food_stores = (food_stores - population) + (food_grow*city);// Food growth increases with city count

```

Comment:
Food isn't grown in cities though, it's grown on country land so I don't thing this should be changed

Reply:
But since cities arent actually taking up land space, we can have it as way to increment food stock nai?